manager.py

```diff
@@ -246,7 +246,32 @@ def askResponse(player, action):
 			player.coin += 2
 
 		if claimedIdentity == "Spy":
-			pass
+			decisionList = ["Y", "N"]
+			if player.order == userOrder:
+				print "Which two player do you wnat to look at and swap?"
+				fstTarget = int(raw_input("Target 1: "))
+				print playerList[fstTarget].actualRole
+				sndTarget = int(raw_input("Target 2: "))
+				print playerList[sndTarget].actualRole
+				decision = raw_input("Do you want to swap two player's card? (Y/N)")
+			else:
+				fstTarget = random.choice(playerList).order
+				sndTarget = random.choice(playerList).order
+				while fstTarget == sndTarget:
+					sndTarget = random.choice(playerList).order
+				fstTarget = int(fstTarget)
+				sndTarget = int(sndTarget)
+				decision = random.choice(decisionList)
+
+			if decision == "N":
+				print ">>> Not swap anyone! <<<"
+				pass
```

Comment:
Is there any possible reason that one player announce the spy's ability but do nothing? If so, when and why?